src/GeniusYield/GYConfig.hs

```diff
@@ -138,10 +138,10 @@ In JSON format, this essentially corresponds to:
 = { coreProvider: GYCoreProviderInfo, networkId: NetworkId, logging: [GYLogScribeConfig], utxoCacheEnable: boolean }
 -}
 data GYCoreConfig = GYCoreConfig
-  { cfgCoreProvider    :: !GYCoreProviderInfo
-  , cfgNetworkId       :: !GYNetworkId
-  , cfgLogging         :: ![GYLogScribeConfig]
-  , cfgUtxoCacheEnable :: !Bool
+  { cfgCoreProvider :: !GYCoreProviderInfo
+  , cfgNetworkId    :: !GYNetworkId
+  , cfgLogging      :: ![GYLogScribeConfig]
+  -- , cfgUtxoCacheEnable :: !Bool
```

Comment:
Our earlier config files would still be valid as this variable (if mentioned in `json` file) would simply be ignored for.